features/xacml/org.wso2.carbon.identity.xacml.server.feature/resources/org.wso2.carbon.identity.xacml.server.feature.default.json

```diff
@@ -15,9 +15,9 @@
   "identity.entitlement.policy_point.pdp.registry_level_policy_cache_clear": false,
   "identity.entitlement.policy_point.pdp.reference_max_policy_entries": "3000",
   "identity.entitlement.policy_point.pdp.policy_finders": [
-    "org.wso2.carbon.identity.entitlement.dao.RegistryPolicyDAOImpl"
+    "org.wso2.carbon.identity.entitlement.dao.RegistryPolicyPersistenceManager"
```

Comment:
Having this as registry would lead to use registry impl in a fresh setup right? Shall we have db impl as the default value here?